.travis.yml

```
@@ -50,6 +24,9 @@ matrix:
     - php: 7.4
       env:
         - DEPS=latest
+    - php: 8.0
+      env:
+        - DEPS=latest
```

Please `nightly` instead of `8.0`:
```suggestion
    - php: nightly
      env:
        - DEPS=lowest
    - php: nightly
      env:
        - DEPS=latest
```